src/components/landing-tree-status/LandingTreeStats.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
Folder name should be landing-tree-stats I assume?

Reply:
Yeah, changing that now

Reply:
Let's stick with camelCase - landingTreeStatus. Generally, easier to type and read IMO.